usaon_vta_survey/forms.py

```diff
@@ -20,11 +25,20 @@
     ResponseDataProductApplication,
     ResponseObservingSystem,
     ResponseObservingSystemDataProduct,
+    Role,
     Survey,
     User,
 )
 
 
+def possible_roles():
+    return Role.query
+
+
+class RoleEdit(Form):
+    role_list = QuerySelectField(query_factory=possible_roles)
```

Comment:
Since we're not implementing the drop-down yet, I think we should remove the above function and class!